openpathsampling/netcdfplus/dictify.py

```diff
@@ -205,7 +211,7 @@ def to_json(self, obj, base_type=''):
         return json.dumps(simplified)
 
     def to_json_object(self, obj):
-        if hasattr(obj, 'base_cls'):
+        if hasattr(obj, 'base_cls') and type(obj) is not type and type(obj) is not abc.ABCMeta:
```

Comment:
When do you have `type(obj) is abc.ABCMeta` while trying to store the object? My intuition would have been that it was impossible for that to occur, since `abc.ABCMeta` shouldn't get instantiated.